OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -1051,9 +1048,8 @@ void Nudge(Actor self)
 				.Rotate(WRot.FromFacing(self.World.SharedRandom.Next(256)));
 			var target = Target.FromPos(self.CenterPosition + offset);
 
-			self.CancelActivity();
```

Comment:
Intentional? If yes, I think this would be better in a separate commit with a short explanation, as this doesn't really have to do with the target lines.